apidoc

```diff
@@ -50,5 +50,7 @@ $application = new yii\console\Application([
 if ($vendorPath !== null) {
     $application->setVendorPath($vendorPath);
 }
+Yii::setAlias('@bower', '@vendor/bower-asset');                                 
```

Comment:
I think, it should be better set via config array.

Reply:
@ricpelo aha, I see, setVendorPath() overrides it anyway

Reply:
this breaks it for all cases where its a different directory name. I think its better to go the same way as with composer above and check whether either `bower` or `bower-asset` exist and use the one that is available.